app/grandchallenge/challenges/models.py

```diff
@@ -1402,3 +1403,81 @@ class ChallengeRequestGroupObjectPermission(GroupObjectPermissionBase):
     content_object = models.ForeignKey(
         ChallengeRequest, on_delete=models.CASCADE
     )
+
+
+class TaskResponsibleChoices(models.TextChoices):
+    SUPPORT = "SUP", "Support"
+    CHALLENGE_ORGANIZERS = "ORG", "Challenge Organizers"
+
+
+class OnboardingTask(FieldChangeMixin, UUIDModel):
+    ResponsibleChoices = TaskResponsibleChoices
+
+    class Meta:
+        permissions = [
+            ("complete_onboaringtask", "Can mark this task as completed")
+        ]
+
+    created = models.DateTimeField(editable=False)
```

Comment:
Add this field without the `auto_now_add` as to have `completed_at` and `created_at` match perfectly when objects are created with `complete=True`.

Reply:
This sort-of stemmed from the idea of us looking back at what the resolve time is of the tasks in the future. I added a corner-case in a test and my inner nerd wanted to get that perfect. A rational for reverting it would be that it adds a small amount of complexity.

Adding the `at` would not have it overwrite the parent class, so I'd then need to drop the parent class and add uuid etc myself. That is not something I am fond of doing.

Reply:
I imagined some BI in the future, calculating the resolve time for tasks and finding some in `ms` range. Only to find out that happens when a task was created with a `complete=True`. Then having to filter, oddly, on very low resolve time instead of being able to exclude via `created_is==completed_at`.

Reply:
Ah, got it! I'm still wondering why we would ever want to add tasks that are already completed, but if there is a use case for that, then I agree that it would be nice for those tasks to have `created==completed_at.`